dashboard/requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy==1.19.2
 altair==4.1.0
-streamlit==0.81.0
```

Comment:
Ein Pull Request sollte immer möglichst nur eine inhaltliche Änderung enthalten. Zukünftig wäre es besser, ein Update der Abhängigkeiten in einen separaten PR zu packen.